lib/ansible/plugins/lookup/passwordstore.py

```diff
@@ -35,20 +38,21 @@ def check_output2(*popenargs, **kwargs):
     if 'input' in kwargs:
         if 'stdin' in kwargs:
             raise ValueError('stdin and input arguments may not both be used.')
-        inputdata = kwargs['input']
+        inputdata = to_bytes(kwargs['input'])
         del kwargs['input']
         kwargs['stdin'] = subprocess.PIPE
     else:
         inputdata = None
     process = subprocess.Popen(*popenargs, stdout=subprocess.PIPE, stderr=subprocess.PIPE, **kwargs)
     try:
         out, err = process.communicate(inputdata)
+        out, err = to_native(out), to_native(err)
     except:
         process.kill()
         process.wait()
         raise
     retcode = process.poll()
-    if retcode:
+    if retcode != 0 or 'encryption failed: Unusable public key' in out + err:
```

Comment:
any() isn't the most readable way to write this, but it's clearer what's intended than ```"here's a message" in out + err```.  The latter could be checking whether out == "Toshio, here's a" and err == "message end".

I'm okay with this not being the fastest code but being more clear.  how about the long hand:
```
     if (retcode != 0 or 'encryption failed: Unusable public key' in out or
         'encryption failed: Unusable public key' in err): 
```